spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -1,6 +1,6 @@
 management:
   security:
-    roles: ADMIN
```

Comment:
@sabbyanandan @trisberg Can we just remove this property altogether? It seems to create issues when allocating roles - It looks like the management role is merged into the user roles, making it hard to assign default roles. Or even to create a user without a Manage role.